Ajout d'un entête

JesseEmond opened the pull request:
Ajout d'un entête sur desktop & mobile, avec le logo de VireAuVert qui pointe vers https://www.vireauvert.org/

Visuellement:

---

## Mobile

![image](https://user-images.githubusercontent.com/1843555/187124104-bd4bb8c2-cc1e-4c08-ad40-ef7c7affd39e.png)


---

## Desktop

![image](https://user-images.githubusercontent.com/1843555/187123661-294c656c-03ec-46fe-99dd-7842041d8435.png)


---

Commit:
WIP: header

Commit:
Merge branch 'main' into header

Commit:
Merge branch 'main' into header

Also add clickable logo and some styling (ideally should have been as
separate commit).

Commit:
Alignement du logo dans le header avec legende

Commit:
Fix logo img src path

Commit:
Touche visuelle

Drahakar commented:
sans gutter c'est un peu intense pour le reste de l'application. possible de le rajouter pour le timeline?

Commit:
Merge branch 'main' into header

Commit:
wip header fix

Commit:
Merge branch 'main' into header

Commit:
Merge branch 'main' into header

Commit:
Simplification du header sur desktop
diff --git a/src/App.vue b/src/App.vue
index d3b639c..5d84abf 100644
--- a/src/App.vue
+++ b/src/App.vue
@@ -61,9 +61,9 @@ Sentry.init({
 </script>

 <template>
-    <div id="main" class="mx-0">
-        <DesktopApp v-if="isDesktop" class="desktop-layout" :userState="state"></DesktopApp>
-        <MobileApp v-else class="mobile-layout" :userState="state"></MobileApp>
+    <div id="main">
+        <DesktopApp v-if="isDesktop" :userState="state"></DesktopApp>
+        <MobileApp v-else class="mx-0" :userState="state"></MobileApp>
         <div id="loading-overlay" v-if="!loadingCompleted">
             <div class="spinner-border" role="status" style="width: 5rem; height: 5rem;"></div>
             <h3 class="mt-2">Chargement des données...</h3>
@@ -93,8 +93,6 @@ Sentry.init({

 <style>
 /* global */
-.mobile-layout {}
-.desktop-layout {}
 .tab-panel {
     padding-top: 10px;
     height: 100%;
diff --git a/src/DesktopApp.vue b/src/DesktopApp.vue
index 382e627..507cfeb 100644
--- a/src/DesktopApp.vue
+++ b/src/DesktopApp.vue
@@ -69,10 +69,10 @@ export default defineComponent({
 </script>

 <template>
-    <div class="d-md-block desktop-layout">
-        <Header class="header col-12"></Header>
-        <div class="row g-0">
+    <div class="container-fluid d-md-block">
+        <div class="row">
             <div class="legend col-md-3">
+                <Header class="header"></Header>
                 <RegionSearch :district="userState.district" @district-selected="selectDistrict"></RegionSearch>
                 <Statistics :district="userState.district" :year="userState.year"></Statistics>
                 <CandidateList :district="userState.district"></CandidateList>
@@ -84,7 +84,7 @@ export default defineComponent({
                 :catastrophes="catastrophes" @district-selected="selectDistrict" :location="userState.location"
                 @location-changed="mapMoved" :zoom="userState.zoom" @zoom-changed="mapZoomed"></MapView>
         </div>
-        <div class="row g-0">
+        <div class="row">
             <Timeline class="timeline" :year="userState.year" @year-selected="selectYear" :district="userState.district"></Timeline>
         </div>
     </div>
@@ -92,13 +92,12 @@ export default defineComponent({

 <style scoped>
 .header {
-    height: 64px;
-    margin-bottom: 8px;
+    height: 128px;
 }

 .map-view {
     min-height: 400px;
-    height: calc(100vh - 96px - 64px - 8px);
+    height: calc(100vh - 96px);
 }

 .legend {
@@ -106,7 +105,7 @@ export default defineComponent({
     flex-direction: column;
     gap: 10px;
     padding: 10px;
-    max-height: calc(100vh - 96px - 64px - 8px);
+    max-height: calc(100vh - 96px);
 }

 .timeline {
diff --git a/src/components/Header.vue b/src/components/Header.vue
index d78749c..c3ddc19 100644
--- a/src/components/Header.vue
+++ b/src/components/Header.vue
@@ -1,13 +1,8 @@
 <template>
     <div class="header">
-        <div class="header-logo flex-1">
-            <a href="https://www.vireauvert.org/" target="_blank">
-                <img class="logo" src="/VAV_2022.png">
-            </a>
-        </div>
-        <div class="header-info flex-1">
-            <!-- TODO: call to action? -->
-        </div>
+        <a href="https://www.vireauvert.org/" target="_blank">
+            <img class="logo" src="/VAV_2022.png">
+        </a>
     </div>
 </template>

@@ -26,19 +21,10 @@ export default defineComponent({
     display: flex;
     flex-direction: row;
     justify-content: center;
-    align-items: center;
-}
-
-.header-logo {
-    min-width: 25%;
-    height: 100%;
-}
-
-.header-info {
-    flex: 1;
 }

 .logo {
     max-height: 100%;
 }
+
 </style>

Commit:
Simplifications apres changements

Commit:
Merge branch 'main' into header

Commit:
Retrait de changements inutiles dans la PR

JesseEmond commented:
> sans gutter c'est un peu intense pour le reste de l'application. possible de le rajouter pour le timeline?

oui ce n'est plus nécessaire maintenant, vu que le header en ce moment à le même background que le reste de l'application je n'ai plus besoin de me battre avec le gutter, donc j'ai enlevé ça.

Commit:
Alignement à gauche du logo sur mobile
